samples/kafka_redis_adapter/docker-compose.l4t.yml

```diff
@@ -0,0 +1,130 @@
+version: "3.3"
+
+services:
+  kafka:
+    image: bitnami/kafka:3.5.1
+    restart: unless-stopped
+    volumes:
+      - "kafka_data:/bitnami"
+    environment:
+      # KRaft settings
+      - KAFKA_CFG_NODE_ID=0
+      - KAFKA_CFG_PROCESS_ROLES=controller,broker
+      - KAFKA_CFG_CONTROLLER_QUORUM_VOTERS=0@kafka:9093
+      # Listeners
+      - KAFKA_CFG_LISTENERS=PLAINTEXT://:9092,CONTROLLER://:9093
+      - KAFKA_CFG_ADVERTISED_LISTENERS=PLAINTEXT://:9092
+      - KAFKA_CFG_LISTENER_SECURITY_PROTOCOL_MAP=CONTROLLER:PLAINTEXT,PLAINTEXT:PLAINTEXT
+      - KAFKA_CFG_CONTROLLER_LISTENER_NAMES=CONTROLLER
+      - KAFKA_CFG_INTER_BROKER_LISTENER_NAME=PLAINTEXT
+    healthcheck:
+      test:
+        - CMD-SHELL
+        - >
+          /opt/bitnami/kafka/bin/kafka-topics.sh --bootstrap-server kafka:9092 --topic hc --create --if-not-exists
+          && /opt/bitnami/kafka/bin/kafka-topics.sh --bootstrap-server kafka:9092 --topic hc --describe
+      start_period: 10s
+      interval: 10s
+      timeout: 5s
+      retries: 5
+
+  keydb:
+    image: eqalpha/keydb:arm64_v6.3.3
+    restart: unless-stopped
+
+  video-loop-source:
+    image: ghcr.io/insight-platform/savant-adapters-gstreamer-l4t:latest
+    restart: unless-stopped
+    volumes:
+      - zmq_sockets:/tmp/zmq-sockets
+      - /tmp/video-loop-source-downloads:/tmp/video-loop-source-downloads
+    environment:
+      - LOCATION=https://eu-central-1.linodeobjects.com/savant-data/demo/Free_City_Street_Footage.mp4
+      - DOWNLOAD_PATH=/tmp/video-loop-source-downloads
+      - ZMQ_ENDPOINT=pub+bind:ipc:///tmp/zmq-sockets/loop-source.ipc
+      - SOURCE_ID=city-traffic
+      - SYNC_OUTPUT=True
+    entrypoint: /opt/savant/adapters/gst/sources/video_loop.sh
+    depends_on:
+      module:
+        condition: service_healthy
+      kafka-redis-sink:
+        condition: service_started
+
+  kafka-redis-sink:
+    image: ghcr.io/insight-platform/savant-adapters-py-l4t:latest
+    restart: unless-stopped
+    volumes:
+      - zmq_sockets:/tmp/zmq-sockets
+    environment:
+      - ZMQ_ENDPOINT=sub+connect:ipc:///tmp/zmq-sockets/loop-source.ipc
+      - KAFKA_BROKERS=kafka:9092
+      - KAFKA_TOPIC=kafka-reds-adapter-demo
+      - KAFKA_CREATE_TOPIC=True
+      - 'KAFKA_CREATE_TOPIC_CONFIG={"num_partitions": 4, "replication_factor": 1}'
+      - REDIS_HOST=keydb
+      - REDIS_PORT=6379
```

Comment:
@tomskikh Need TTL setting

Reply:
What about Redis DB?

Reply:
Added TTL and DB settings to the sample.